components/Layout/MainNavigation/mobileView.tsx

```diff
@@ -7,6 +7,8 @@ import twit from "public/images/icons/twitter-blue.svg";
 import face from "public/images/icons/facebook-blue.svg";
 import link from "public/images/icons/linkedinIcon.webp";
 import classes from "./navigation.module.css";
+import { Subscribe } from "@/components/Subscribe";
+import { BlurWrapper } from "@/components/BlurrWrapper";
```

Comment:
Why is BlurWrapper being imported? Don't see it being used in component, maybe I am missing it?

Reply:
Also, left over comments that are not necessary anymore can be deleted before pushing to development branch.

Reply:
BlurWrapper was imported by mistake. Just deleted it. About the comments - all these comments were written by Slavo, so I'm unsure about deleting them, maybe he'll need them at some point? What do you think, should I leave them or not?